rust/lance-index/Cargo.toml

```diff
@@ -95,6 +98,12 @@ pprof.workspace = true
 # docs.rs uses an older version of Ubuntu that does not have the necessary protoc version
 features = ["protoc"]
 
+[profile.release]
```

Comment:
Elsewhere, `debug = true` is only on `release-with-debug` and `bench`, not `release`.